src/js/components/githubUsers/assignTaskRole.tsx

```diff
@@ -129,6 +129,29 @@ const AssignTaskRoleModal = ({
   const alertType =
     orgType === ORG_TYPES.DEV ? 'should_alert_dev' : 'should_alert_qa';
 
+  const footerComponents = (
+    window.GLOBALS.EMAIL_ENABLED
+      ? [
+          <Checkbox
+            key="alert"
+            labels={{ label: checkboxLabel }}
+            className="slds-float_left slds-p-top_xx-small"
+            name={alertType}
+            checked={shouldAlertAssignee}
+            onChange={handleAlertAssignee}
+          />,
+        ]
+      : []
```

Comment:
↔️ : I know we use prettier to avoid this kind of review comment, but I apparently can't help myself: I find multiline ternaries way harder to read.